plugin/trino-hive/src/main/java/io/trino/plugin/hive/HivePartitionManager.java

```diff
@@ -176,22 +186,35 @@ public HivePartitionResult getPartitions(ConnectorTableHandle tableHandle, List<
         return partitionList.build();
     }
 
-    public HiveTableHandle applyPartitionResult(HiveTableHandle handle, HivePartitionResult partitions, Optional<Set<ColumnHandle>> columns)
+    public HiveTableHandle applyPartitionResult(HiveTableHandle handle, HivePartitionResult partitions, Constraint constraint)
     {
+        Optional<List<HivePartition>> partitionList = Optional.empty();
+        Optional<List<String>> partitionNames = partitions.getPartitionNames();
```

Comment:
partitionNames always go before partitionList: reorder the variable declarations, and assignments below